crates/model/src/orders/stop_limit.rs

```diff
@@ -58,8 +64,12 @@ pub struct StopLimitOrder {
 
 impl StopLimitOrder {
     /// Creates a new [`StopLimitOrder`] instance.
+    ///
+    /// # Errors
+    ///
+    /// Returns an error if the order is invalid.
```

Comment:
added this because got this error [here](https://github.com/nautechsystems/nautilus_trader/actions/runs/14813508656/job/41591161735)
<img width="654" alt="image" src="https://github.com/user-attachments/assets/498f0afd-b679-4760-b8e4-4242687085c9" />